docs/how-to/logical-replication/set-up-clusters.md

```diff
@@ -0,0 +1,104 @@
+# How to set up clusters for logical replication
+
+For testing purposes, you can deploy two applications of the [data integrator charm](https://charmhub.io/data-integrator) and then relate them to the two PostgreSQL clusters you want to replicate data between.
+```sh
```

Comment:
Let's add here:
```
juju deploy postgresql --channel 16/stable postgresql1 # revision rev863+
juju deploy postgresql --channel 16/stable postgresql2 # revision rev863+
```

it will make this page completed. WDYT?